examples/monitors/metric-alert/payload.json

```diff
@@ -0,0 +1,28 @@
+{
+    "tag": "prod-us-west",
+    "agg_func": "sum",
+    "agg_window": "1h",
+    "lookback": "12h",
+    "rule": "api_access",
+    "stat_types": [
+        "count"
+    ],
+    "measurement": "api_access.count",
+    "source_filters": {
+        "sourcetype": "K8s"
+    },
+    "group_bys": [
+        "k8s_namespace"
+    ],
+    "anomaly_greater_than": 60,
+    "anomaly_less_than": 80,
+    "value_greater_than": 10,
+    "value_less_than": 40,
```

Comment:
I know I mentioned let's fill all fields but I should have been more specific. Having all these threshold related fields set together doesn't make sense. Let's just use  anomaly_greater_than: 95 and remove other 3 fields

Reply:
I've actually put all the values just to populate the payload; changed the payload in cc2d024